modules/odc_con_customizing-a-developer-catalog-or-its-sub-catalogs.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: CONCEPT
+
+[id="odc_con_customizing-a-developer-catalog-or-its-sub-catalogs_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs
+
+The Red Hat OpenShift Container Platform Web Console provides support for the Administrator and the Developer perspectives. Cluster administrators have the ability to organize and manage a Developer catalog or its sub-catalogs based on the access control and privileges provided to a user. In some situations, customers do not want their users to have access to a particular Developer catalog or it's sub-catalogs. Customizing a user's role ensures that the user only views the Developer catalog or sub-catalogs resources assigned to them.
+
+For example, if you do not want a user to access Developer catalog resources such as Applications, Operator Backed services, Databases, Helm chart repositories etc, you can customize a user's profile accordingly.
```

Comment:
```suggestion
For example, if you do not want a user to access Developer catalog types such as Templates, Operator Backed services, Databases, Helm Charts, etc, you can define the state of the types as disabled.
```

Reply:
@debsmita1 - Can I remove this sentence entirely since you suggested earlier that "user's role is not considered to Enable/Disable the developer catalog or its sub-catalog types."

Reply:
Fixed.